packages/website/components/account/paymentTable.js/paymentTable.js

```diff
@@ -72,11 +70,12 @@ const PaymentTable = ({ plans, currentPlan, isEarlyAdopter, setPlanSelection, se
                     </Button>
                   )}
 
-                  {(currentPlan?.id === plan.id || (isEarlyAdopter && plan.id === 'earlyAdopter')) && (
```

Comment:
@cmunns It didn't seem we needed to check both `isEarlyAdopter` and `plan.id === 'earlyAdopter'`, but I could be missing something.

Reply:
ah,,,, you're right.